src/wp-a11y-day/functions.php

```diff
@@ -267,6 +267,22 @@ function wp_accessibility_day_widgets_init() {
 }
 add_action( 'widgets_init', 'wp_accessibility_day_widgets_init' );
 
+/**
+ * Register block styles.
+ *
+ * @return void
+ */
+function wp_accessibility_day_block_styles() {
+    register_block_style(
```

Comment:
For consistency, can you register this in /js/block.js? It could get confusing if we're registering block styles in multiple places.

Reply:
@joedolson updated.